### PR TITLE
feat(3000): Add sites sidebar for Toolbar launching

### PR DESCRIPTION
**Pull request by Twixes**
## Changes

It wasn't possible to launch the Toolbar in PostHog 3000. With the new "Sites" feature, it is. You can still open your sites in a new tab, but also view them inside PostHog, with the toolbar on.

Screenshot 2023-07-05 at 14 09 49

Of course with iframes there are significant security considerations, and most importantly most web apps don't allow iframing by default. There's currently no education on resolving this in the app, but we can add it later on.

### CODE REVIEW

File: frontend/src/scenes/scenes.ts [hunk 1]
```diff
@@ -210,12 +210,17 @@ export const sceneConfigurations: Partial<Record<Scene, SceneConfig>> = {
     },
     [Scene.Ingestion]: {
         projectBased: true,
-        plain: true,
+        plain: 2,
```
Comment (thmsobrmlr) [inline frontend/src/scenes/scenes.ts:213]: Can we add an enum for the different states?

**Review by thmsobrmlr — CHANGES_REQUESTED** [review]
I really like the site concept! Code looks solid, one suggestion to add an enum.

Functionality wise I'm still missing being able to add other URLs besides the suggested ones and to edit the existing ones. Most importantly I can't launch the toolbar for local development e.g. to disable posthog-3000.

File: frontend/src/scenes/scenes.ts (hunk 1 above)
Comment (Twixes) [reply frontend/src/scenes/scenes.ts]: Hmm, enum might be overkill, but made this into variants. Take a look now

**Comment by Twixes** [comment]
Good feedback about the lack of "New site" button being annoying, I have a branch in progress that adds that. Assuming that PR is the follow up to this one, do you think this is going to be good enough for now?

**Review by thmsobrmlr — APPROVED** [review]
Yep, good enough to go in. It's adding more functionality and I'm just complaining about missing things :)

**Comment by Twixes** [comment]
BTW it is still possible to launch the site with toolbar in a new tab like in the classic navigation, it's in item menu. Maybe it was not clear that that includes the Toolbar, so renamed from "Open in new tab" to "Open with Toolbar in new tab". Alternatively maybe for important actions like that there could be an action button on items, which in this case would make "Open with Toolbar in new tab" prominent. (@corywatilo)

**Pull request merged**